test_runner/src/test/kotlin/ftl/config/BugsnagInitHelperTest.kt

```diff
@@ -0,0 +1,70 @@
+package ftl.config
+
+import ftl.log.LogbackLogger
+import io.mockk.every
+import io.mockk.mockkStatic
+import io.mockk.unmockkAll
+import org.junit.After
+import org.junit.Assert.assertNotNull
+import org.junit.Assert.assertNull
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.junit.rules.TemporaryFolder
+import java.io.File
+import java.util.UUID
+
+private const val GSUTIL_FOLDER = ".gsutil"
+private const val ANALYTICS_FILE = "analytics-uuid"
+private const val DISABLED = "DISABLED\n"
```

Comment:
Why is there a new line at the end?

Reply:
This file can be added manually by the user. Since some of text editors add new line char at the end of text file automatically I wanted to simulate that case.